content/docs/install/completion.md

```diff
@@ -49,25 +50,24 @@ install and configure shell completion.
    $ brew install bash-completion
    ```
 
-2. Then download the
-   [DVC completion script](https://github.com/iterative/dvc/blob/master/scripts/completion/dvc.bash)
-   and save it on `/usr/local/etc/bash_completion.d/`:
-
-   ```dvc
-   $ sudo wget \
-       -O /usr/local/etc/bash_completion.d/dvc \
-       https://raw.githubusercontent.com/iterative/dvc/master/scripts/completion/dvc.bash
-   ```
-
-3. Edit `~/.bash_profile` and make sure that these lines are there:
+   then edit `~/.bash_profile` and make sure that these lines appear:
 
    ```bash
    if [ -f $(brew --prefix)/etc/bash_completion ]; then
        . $(brew --prefix)/etc/bash_completion
    fi
    ```
 
-4. Finally, `source ~/.bash_profile` or launch a new terminal to activate it.
+2. Run these commands to install DVC completions:
+
+   ```dvc
+   $ DVC_BIN="$(which dvc)"
```

Comment:
This is not constant though and might break if you install dvc from a different source, or if your shim no longer works after you've uninstalled dvc in that env.

Reply:
well it is a constant in the sense that the value of `$(which dvc)` is hard-coded into `~/.bash_completion`.

Also nothing will break if the env is uninstalled as `which` is also used in the `~/.bash_completion` file.

Reply:
Ok, let's keep it for now.